gaitmap_mad/gaitmap_mad/stride_segmentation/hmm/_hmm_feature_transform.py

```diff
@@ -64,7 +65,9 @@ def transform(
         """
         raise NotImplementedError()
 
-    def inverse_transform_state_sequence(self, state_sequence: np.ndarray, *, sampling_rate_hz: float) -> np.ndarray:
+    def inverse_transform_state_sequence(
+        self, state_sequence: np.ndarray, data: np.array, *, sampling_rate_hz: float
```

Comment:
Can you unify the call signature of the base class with your implementaiton? The actual implementation does not get sampling_rate_hz as parameter. So remove it here as well